pkg/logp/logp.go

```diff
@@ -0,0 +1,21 @@
+package logp
```

Comment:
Why did you call this package `logp`?

Reply:
Didn't want to have a conflict with `log` packages. Do you have a different name in mind?

Reply:
Do we have any use for log package? We can always import the conflicting packages under different name.

Reply:
There was not. Was deprioritized.

Reply:
Renamed it to `log`: hope that pleases the crowds :)